src/controllers/baseHandler.ts

```diff
@@ -26,6 +28,18 @@ export async function baseHandler(
         env
       );
     }
+    case getCommandName(MENTION_EACH): {
+      return await mentionEachUser(
+        {
+          //  assertion was added because the options type is optional also eslint had hard rules set so had to cross the border :)
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          displayType: message.data?.options![0]?.name,
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          options: message.data?.options![0]?.options![0]?.options,
```

Comment:
any specific reason for disabling linters?

Reply:
have mentioned in the comment itself